dynawo/sources/Models/CPP/ModelNetwork/DYNModelStaticVarCompensator.cpp

```diff
@@ -88,7 +88,7 @@ stateModified_(false) {
   double ur0 = 0.;
   double ui0 = 0.;
   double U0 = 0.;
-  double Q0 = svc->getQ() / SNREF;
+  double Q0 = svc->getQ();
```

Comment:
I don't think you should delete the division by SNREF here. The getQ() method that you call is the one from DYNStaticVarCompensatorInterfaceIIDM that returns a reactive power in SI units. It is then used to calculate the initial B0 with the following equation: B0 = -Q0/U0² with U0 and B0 being in p.u.. As such, Q0 should be in p.u., except if I have missed a change somewhere.

Reply:
ok right :)